### ISSUE TITLE
[Errata] Chapter 2, Question 7, Review Questions

### ISSUE THREAD

**Issue by indrayam** [awsccp/awsccp.github.io#67]
Chapter 2. Review Question 7
The question was "Which of the following AWS documentation URLs points to the page containing an up-to-date list of service limits?". 

My comments:
- Do they *really* ask for correct URLs in this day and age? I mean these things can change on a dime.
- If we are indeed going to ask for the URL that contains an up-to-date list of service **quotas** (they seem to have deprecated the use of the word "limits" and replaced it with quotas), then the correct URL should be [Service endpoints and quotas](https://docs.aws.amazon.com/general/latest/gr/aws-service-information.html)

**Comment by dbclinton** [comment]
You're correct that they probably won't ask for a specific URL (or, at least, I can't remember getting a question like that). But this specific question is really designed to help you with your study and research: Amazon is pretty good at maintaining useful and consistent URL conventions for this kind of thing.
Having said that, we're probably all using ChatGPT at this point anyway, right?

**Comment by indrayam** [comment]
You can say that again 👍🏼
